themes/src/main/resources-community/theme/base/admin/messages/messages_fr.properties

```diff
@@ -7,3 +7,12 @@ invalidPasswordNotUsernameMessage=Mot de passe invalide : ne doit pas être iden
 invalidPasswordRegexPatternMessage=Mot de passe invalide : ne valide pas l''expression rationnelle.
 invalidPasswordHistoryMessage=Mot de passe invalide : ne doit pas être égal aux {0} derniers mot de passe.
 invalidPasswordMaxLengthMessage=Mot de passe invalide : longueur maximale acceptée à {0}.
+ldapErrorMissingClientId=L'ID du client doit être renseigné dans le configuration quand l'association des rôles du domaine n'est pas utilisé.
+invalidPasswordNotContainsUsernameMessage=Mot de passe invalide : il ne doit pas contenir le nom d'utilisateur.
+invalidPasswordNotEmailMessage=Mot de passe invalide : il ne doit pas correspondre à l'adresse de courriel.
+invalidPasswordBlacklistedMessage=Mot de passe invalide : mot de passe blacklisté.
+invalidPasswordGenericMessage=Mot de passe invalide : le nouveau mot de passe ne respecte pas les politiques de mot de passe.
+ldapErrorEditModeMandatory=Le mode d'édition est obligatoire
```

Comment:
Thank you for this translation, @Dodouce!

Once thing to note is that for all backend messages, all single quotes need to be duplicated, while for the frontend they must not be. 

See [the docs](https://github.com/keycloak/keycloak/blob/main/docs/translation.md#handling-of-single-quotes-)  for a more detailed explanation where to use what. 

The automatic check flagged those items, and I will go ahead and fix them in Weblate. For future translations, please try to use the correct double single quotes.

Alternatively, there are typographic quote that will work just normal in the frontend and the backend. Maybe 
 the two language maintainers @jmorille and @blackholegalaxy have an opinion here as well.